randovania/games/prime2/json_data/Sanctuary Fortress.txt

```diff
@@ -2685,7 +2695,7 @@ Extra - in_dark_aether: True
                               Light Suit
                               Suitless Ingclaw/Ingstorm (Intermediate) and Dark World Damage ≥ 50 and Ingstorm Damage ≥ 70
                       All of the following:
-                          Screw Attack at Z-Axis (Intermediate) and Screw Attack into Tunnels/Openings (Expert) and Disabled Room Randomizer
+                          Screw Attack into Tunnels/Openings (Advanced) and Disabled Room Randomizer
```

Comment:
Reason for this removal?

Reply:
Erin said that this is technically not z-axis and should be replaced to Screw Attack into tunnels. If you disagree I'll revert it back

Reply:
The reason you can even do this nsj is because of the z-axis. In room rando this wouldn't be possible.